dev_setup.sh

```diff
@@ -137,6 +137,22 @@ This script is designed to make working with Mycroft easy.  During this
 first run of dev_setup we will ask you a few questions to help setup
 your environment.'
     sleep 0.5
+    if ! grep -q avx /proc/cpuinfo; then
```

Comment:
Does this condition apply to the ARM platform as well? Since we have a precise build for that platform it might need to be excluded.

Reply:
Didn't know about the ARM exception. Looks like that can be easily checked with  $(uname -m) == 'arm'* so I've added that to the logic

Reply:
Not easy to know :)

That looks like a good solution, great work with that.

Reply:
Hey, we had some chat about this [on the Forums](https://community.mycroft.ai/t/running-mycroft-on-an-arm-cpu-without-avx/7056/4?u=gez-mycroft). According to @domcross the ARM equivalent is called SIMD (Single Instruction, Multiple Data) and is available since ARMv7 (which includes the Cortex A53 used by the RaspberryPi and the Cortex A7 from the OrangePi).
Is it worth checking for SIMD on ARM?

Reply:
The arm equivalents seem to have a bunch of different names. on the Pi it's *neon* and on the H64 it's called *asimd* so it might not be trivial to do a correct general check.

Reply:
Ah, good to know, ignore me then :) 
I think there's much less presumption that it will work on random ARM hardware anyway.